packages/postcss-merge-longhand/src/lib/canExplode.js

```diff
@@ -1,12 +1,11 @@
 import isCustomProp from './isCustomProp';
 
-const isInherit = node => ~node.value.indexOf('inherit');
-const isInitial = node => ~node.value.indexOf('initial');
+const isReserved = node => ["inherit", "initial", "unset"].includes(node.value);
 
 export default (prop, includeCustomProps = true) => {
     if (includeCustomProps && isCustomProp(prop)) {
         return false;
     }
 
-    return !isInherit(prop) && !isInitial(prop);
+    return !isReserved(prop);
```

Comment:
What makes `inherit` and `initial` "reserved"`?

Reply:
I agree it is a bit vague, this is what this is referred to in other areas of this codebase. Since it's only called once I can just inline the includes and avoid naming this.